middlewares/hacker.middleware.js

```diff
@@ -129,13 +129,11 @@ function addDefaultStatus(req, res, next) {
 }
 
 /**
- * Verifies that account is confirmed and of proper type from the account ID passed in req.body.accountId
- * @param {{body: {accountId: ObjectId}}} req 
- * @param {*} res 
+ * Helper function that validates if account is confirmed and is of proper type
+ * @param account account object containing the information for an account 
  * @param {(err?) => void} next 
  */
-async function validateConfirmedStatus(req, res, next) {
-    const account = await Services.Account.findById(req.body.accountId);
+async function validateConfirmedStatus(account, next) {
```

Comment:
does change in req, res, next structure affect how we chain the middleware?

Reply:
also is this needed? if we put the if statement in validateConfirmedStatusFromAccountId does it still work?

Reply:
Made them separated as we have 3 different validateConfirmed statuses: fromAccountID, fromHackerID and fromObject